pkg/kubelet/kubelet.go

```diff
@@ -3083,10 +3082,10 @@ func (kl *Kubelet) cleanUpContainersInPod(podID types.UID, exitedContainerID str
 // Function is executed only during Kubelet start which improves latency to ready node by updating
 // kubelet state, runtime status and node statuses ASAP.
 func (kl *Kubelet) fastStatusUpdateOnce() {
-	ctx := context.Background()
 	start := kl.clock.Now()
 	stopCh := make(chan struct{})
-
+	ctx, cancel := context.WithTimeout(context.Background(), nodeReadyGracePeriod)
```

Comment:
The `fastStatusUpdateOnce` function can receive a context parameter to replace `context.Background()`. This should be harmless since the context generated by `kubelet.Run()` is consistent with the lifecycle of the kubelet process. Moreover, you have derived a child context from it, so it will not affect CRI or other calls.

Reply:
done